test/e2e/network/dns_scale_records.go

```diff
@@ -50,10 +51,22 @@ var _ = SIGDescribe("[Feature:PerformanceDNS]", func() {
 
 	// answers dns for service - creates the maximum number of services, and then check dns record for one
 	It("Should answer DNS query for maximum number of services per cluster", func() {
-		services := generateServicesInNamespace(f.Namespace.Name, maxServicesPerCluster)
+
+		numNs := maxServicesPerCluster / maxServicesPerNamespace
```

Comment:
Why not
```
numNs := (maxServicesPerCluster + maxServicesPerNamespace - 1) / maxServicesPerNamespace
```

Reply:
Not +1, but -1.
Then you get:
(10000 + 5000 - 1) / 5000 = 2.99.. which will be 3.

(A+B-1)/B is integer ceiling of A/B